ocs_ci/utility/utils.py

```diff
@@ -2456,7 +2456,16 @@ def get_testrun_name():
     )
     ocp_version_string = f"OCP{ocp_version}" if ocp_version else ""
     ocs_version = config.ENV_DATA.get("ocs_version")
-    ocs_version_string = f"OCS{ocs_version}" if ocs_version else ""
+    fusion_version = config.ENV_DATA.get("fusion_version")
+    fdf_version = config.ENV_DATA.get("fdf_version")
+    if fdf_version:
+        product_version = f"FDF{fdf_version}"
+    elif fusion_version:
+        product_version = f"Fusion{fusion_version}"
+    elif ocs_version:
+        product_version = f"OCS{ocs_version}"
+    else:
+        product_version = ""
```

Comment:
above will skip fusion_version if fdf_version exists. I think we need Fusion and FDF version in subject

Reply:
This was intentional as I didn't want to have both in the name of the launch. If fdf_version exists, I wanted it to take priority over fusion_version.